ambry-api/src/main/java/com/github/ambry/config/ClusterMapConfig.java

```diff
@@ -372,6 +372,11 @@ public class ClusterMapConfig {
   @Default("false")
   public final boolean clusterMapIgnoreDownwardStateTransition;
 
+  public static final String ENABLE_FILE_COPY_FOR_BOOTSTRAP = "clustermap.enable.file.copy.for.bootstrap";
+  @Config(ENABLE_FILE_COPY_FOR_BOOTSTRAP)
+  @Default("false")
+  public final boolean enableFileCopyForBootstrap;
```

Comment:
Keeping the nomenclature in this file consistent, the name of this key would be 
`clusterMapEnableFileCopyForBootstrap`